curriculum/challenges/english/25-front-end-development/lab-event-hub/66ebd4ae2812430bb883c787.md

```diff
@@ -204,8 +204,8 @@ assert.strictEqual(h3Elements.length, 2);
 Both of the `article` elements inside the `#past-events` section should have a paragraph element for the event description.
 
 ```js
-const pElement = document.querySelectorAll('#past-events article h3+p').length;
-assert.strictEqual(pElement, 2);
+const pElements = document.querySelectorAll('#past-events article h3 ~ p');
+assert.lengthOf(pElements, 4);
```

Comment:
the requirement is at least one `p` element for each `article`. This test would fail if one put exactly one `p` in each `article`.
This below checks that there is at least one `p` in each article element, with possibility of more `p`s


```suggestion
const articles = document.querySelectorAll('#past-events article');
articles.forEach(article => {
assert.atLeast(document.querySelectorAll('h3 ~ p').length, 1)
})
```